README.md

```diff
@@ -24,6 +24,11 @@ NEXT_PUBLIC_BLUESKY_CLIENT_URI=https://c5a1-157-131-19-230.ngrok-free.app
 ngrok http http://localhost:3000 --url spicy-rooster-fondly.ngrok-free.app
```

Comment:
it looks like `--domain` works here too - let's just change the docs to use `--domain` since it works for everyone!

Reply:
alright, lemme push that now.